adapters/repos/db/inverted/objects_test.go

```diff
@@ -57,6 +64,26 @@ func TestAnalyzeObject(t *testing.T) {
 				DataType:     []string{"string"},
 				Tokenization: "field",
 			},
+			{
+				Name:         "id1",
+				DataType:     []string{"uuid"},
+				Tokenization: "field",
+			},
+			{
+				Name:         "id2",
+				DataType:     []string{"uuid"},
+				Tokenization: "field",
+			},
+			{
+				Name:         "idArray1",
+				DataType:     []string{"uuid[]"},
+				Tokenization: "field",
+			},
+			{
+				Name:         "idArray2",
+				DataType:     []string{"uuid[]"},
+				Tokenization: "field",
+			},
```

Comment:
is tokenization needed for uuids?

Reply:
Good question. It shouldn't. It's mainly there from copy/pasting. Let me see what happens if I remove it.

Reply:
You were correct, they are not needed. I've removed them in the latest commit.